Allow enabling legacy code in ITK and build its wrappers at the same time.
The goal is to build the native Python wrapper of ITK while making it possible
to have third-party software that still relies on older interfaces.
author: Alex Domingo (Vrije Universiteit Brussel)
--- CMakeLists.txt.orig	2023-12-14 15:26:20.086769773 +0100
+++ CMakeLists.txt	2021-08-12 03:08:23.000000000 +0200
@@ -334,9 +334,9 @@
 #-----------------------------------------------------------------------------
 # Provide compatibility options.
 # During major release updates deprecated interface may be needed for backwards compatibility
-cmake_dependent_option(ITK_LEGACY_REMOVE
-  "Remove current legacy code completely." OFF
-  "NOT ITK_WRAPPING" ON)
+#cmake_dependent_option(ITK_LEGACY_REMOVE
+#  "Remove current legacy code completely." OFF
+#  "NOT ITK_WRAPPING" ON)
 
 # During minor releases bugs may be identified that identify broken interface, or
 # useless interfaces that need to be retained to not break backwards compatibilty.
